src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1567,6 +1560,50 @@ def execute(self, part_df, gens_mega):
         return part_df
 
 
+class AddMaxMinAttribute(AddAttribute):
```

Comment:
Renamed to `AddMaxMinAttribute` since `drop_duplicates` kind of necessitates that you take either the max or min in a grouping. Could go back to `AddSortedAttribute` if there's a more general use case for this or if `drop_duplicates` is reworked, or if this name is confusing.